app/javascript/authentication/SignInForm.tsx

```diff
@@ -139,6 +140,22 @@ const SignInForm = () => {
   )
   const { signIn } = useContext(UserContext)
 
+  const handleRequestError = (error: AxiosError<{ error: string; email: string }>) => {
+    if (error.response.data.error === "not_confirmed") {
+      setNewAccountNotConfirmedModal(error.response.data.email)
+    } else if (error.response.data.error === "bad_credentials") {
+      setRequestError(
+        t("signIn.badCredentialsWithResetLink", {
+          url: getForgotPasswordPath(),
+        })
+      )
+    } else {
+      // TODO: handle sign-in error states
```

Comment:
are there a tickets we can link these TODO's to?

Reply:
Yeah, I am waiting to hear back from design for the 503 one

Reply:
will add 503 error message